apps/docs/src/app/platform/component-docs/platform-forms/radio-group/platform-radio-group-examples/platform-radio-group-list-items-example.component.html

```diff
@@ -1,60 +1,44 @@
-<p>Reactive form Examples</p>
-<form [formGroup]="customForm" class="flex-form">
-    <p>Example 1: Radio Buttons Types with default value and error status</p>
-    <fieldset fd-fieldset>
-        <fdp-radio-group
-            [name]="'radio1'"
-            [value]="'Option 2'"
-            [list]="items"
-            [isInline]="false"
-            formControlName="example1"
-        >
+<h3>Use With Reactive Form</h3>
+
+<p>Platform Radio Buttons Created with given List of SelectItem Objects and have Validation Error</p>
+<fdp-form-group [multiLayout]="true" [formGroup]="form1">
+    <fdp-form-field [id]="'radio1'" zone="zLeft" rank="1">
+        <fdp-radio-group [name]="'radio1'" [value]="'spring'" [list]="items" [isInline]="false"
+            formControlName="example1">
         </fdp-radio-group>
-    </fieldset>
-    <p>Example 2: Radio Buttons Types in Compact contentDensity</p>
-    <fieldset fd-fieldset>
-        <fdp-radio-group
-            [name]="'radio2'"
-            [contentDensity]="'compact'"
-            [list]="items"
-            [isInline]="true"
-            formControlName="example2"
-        >
+    </fdp-form-field>
+</fdp-form-group>
+<p style="margin-left: 2rem;">Selected value: {{ form1.controls.example1.value }}</p>
+
+<p>Compact Platform Radio Button Created with given List of SelectItem Objects.</p>
+<fdp-form-group [multiLayout]="true" [formGroup]="form2">
+    <fdp-form-field [id]="'radio2'" zone="zLeft" rank="1">
+        <fdp-radio-group [name]="'radio2'" [contentDensity]="'compact'" [list]="items" [isInline]="true"
+            formControlName="example2">
         </fdp-radio-group>
-    </fieldset>
-</form>
-<div>
-    Example 1 selelected value: {{ customForm.value.example1 }}
-    <br />
-    Example 2 selelected value: {{ customForm.value.example2 }}
-</div>
+    </fdp-form-field>
+</fdp-form-group>
+<p style="margin-left: 2rem;">Selected value: {{ form2.controls.example2.value }}</p>
```

Comment:
Here we need to add additional example to use DisplayKey and lookupKey we have in CollectionBasedInput. 

You create list of values (complext object) and specify these two properties

Reply:
added example and test case using lookupKey and disaplyKey.